app/controllers/projects_controller.rb

```diff
@@ -23,6 +28,15 @@ def new
 
   # GET /projects/1/edit
   def edit
+    setup_metric_configs(@project)
```

Comment:
I notice if I remove this all the tests still pass, and everything still works - surplus to requirement perhaps